src/test/testing-library/UiDefaultFooter.test.tsx

```diff
@@ -2,13 +2,16 @@ import { render } from '@testing-library/react';
 import React from 'react';
 
 import { DefaultFooter } from '@/components/UiFooter/DefaultFooter';
+import { createTranslation, TranslationFunctionType } from '@/test/translate';
 
 import { mockedSocialLinks } from './constants';
 
+const t: TranslationFunctionType = createTranslation('pages/i18n');
+
 const mockedDate: number = new Date().getFullYear();
 const defaultFooterClass: string = '.MuiStack-root';
-const logoAlt: string = 'Vilna logo';
-const copyright: RegExp = /Copyright/;
+const logoAlt: string = t('footer.logo_alt');
+const copyright: RegExp = new RegExp(t('footer.copyright'));
```

Comment:
for what you use RegExp?

Reply:
regex is used in tests (jest/playwright) when we are not searching the entire text of an element

Reply:
can we without them?

Reply:
Workaround is possible, but I don't quite understand why we should use it if it's a built-in feature of the libraries. Additionally, regex is used in many tests, and coming up with solutions for each case will take time.